quantum/stm32/proton_c.mk

```diff
@@ -3,7 +3,11 @@
 # These are defaults based on what has been implemented for ARM boards
 AUDIO_ENABLE = yes
 RGBLIGHT_ENABLE = no
-BACKLIGHT_ENABLE = no
+
+# Force task driven PWM until ARM can provide automatic configuration
+ifneq ($(strip $(BACKLIGHT_ENABLE)), no)
+	BACKLIGHT_ENABLE = software
```

Comment:
Shouldn't this be "custom"? 

https://github.com/qmk/qmk_firmware/blob/master/common_features.mk#L237

Reply:
This branch doesn't have #6709 merged in, so the option will exist. Hopefully we can get approvals for the other one and then I can rebase this.

Reply:
Aaah, okay.